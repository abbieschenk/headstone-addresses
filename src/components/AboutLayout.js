import React from "react";
import { Link } from "gatsby";

import "../styles/About.scss";

import Layout from "../components/Layout";

const AboutLayout = ({ children }) => {
    return (
        <Layout mainClass="about">
            <div className="about-menu">
                <Link to="/about" className="about-link" activeClassName="active">
                        About
                </Link>
                <Link to="/about/methodology" className="about-link" activeClassName="active">
                        Methodology
                </Link>
            </div>
            <div className="about-background">
                <div className="about-content fade-in" >
                    {children}
                </div>
            </div>
        </Layout>
    )
};

export default AboutLayout;