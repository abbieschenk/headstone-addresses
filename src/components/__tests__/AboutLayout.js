import React from "react"
import renderer from "react-test-renderer"

import AboutLayout from "../AboutLayout"

describe("AboutLayout", () => {
    
  it("renders correctly", () => {
    const tree = renderer
      .create(<AboutLayout><div>Test</div></AboutLayout>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});