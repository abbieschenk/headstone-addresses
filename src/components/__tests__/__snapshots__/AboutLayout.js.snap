// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutLayout renders correctly 1`] = `
Array [
  <header>
    <nav>
      <a
        href="/"
        id="home"
      >
        <div
          id="site-logo"
        >
          <test-file-stub />
        </div>
        <div>
          Headstone Addresses
        </div>
      </a>
      <div
        id="menu"
      >
        <a
          className="nav-link"
          href="/about"
        >
          About
        </a>
        <a
          className="social"
          href="https://instagram.com/headstoneaddresses"
          rel="noreferrer noopener"
          target="_blank"
        >
          <svg
            fill="none"
            height={24}
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            viewBox="0 0 24 24"
            width={24}
            xmlns="http://www.w3.org/2000/svg"
          >
            <rect
              height="20"
              rx="5"
              ry="5"
              width="20"
              x="2"
              y="2"
            />
            <path
              d="M16 11.37A4 4 0 1 1 12.63 8 4 4 0 0 1 16 11.37z"
            />
            <line
              x1="17.5"
              x2="17.51"
              y1="6.5"
              y2="6.5"
            />
          </svg>
        </a>
      </div>
    </nav>
  </header>,
  <main
    className="about"
  >
    <div
      className="about-menu"
    >
      <a
        className="about-link"
        href="/about"
      >
        About
      </a>
      <a
        className="about-link"
        href="/about/contributors"
      >
        Contributors
      </a>
      <a
        className="about-link"
        href="/about/methodology"
      >
        Methodology
      </a>
      <a
        className="about-link"
        href="/about/support"
      >
        Support
      </a>
    </div>
    <div
      className="about-background"
    >
      <div
        className="about-content fade-in"
      >
        <div>
          Test
        </div>
      </div>
    </div>
  </main>,
  <footer>
    Â© Elaine Yip 2021. Supported by the Edmonton Heritage Council and the City of Edmonton.
  </footer>,
]
`;
